modules/offers/client/components/OfferLocationOverlay.js

```diff
@@ -0,0 +1,70 @@
+// External dependencies
+import { BaseControl } from 'react-map-gl';
+import { SVGOverlay } from 'react-map-gl';
```

Comment:
nitpick: imports can be combined
```js
import { BaseControl, SVGOverlay } from 'react-map-gl';
```

Reply:
Oh silly me! We better enable [`no-duplicate-imports`](https://eslint.org/docs/rules/no-duplicate-imports).

Reply:
PRed: https://github.com/Trustroots/trustroots/pull/1306